src/cloudant/client.py

```diff
@@ -157,29 +170,33 @@ def session_cookie(self):
 
         :returns: Session cookie for the current session
         """
-        if self.admin_party:
-            return None
         return self.r_session.cookies.get('AuthSession')
 
     def session_login(self, user=None, passwd=None):
         """
         Performs a session login by posting the auth information
         to the _session endpoint.
+
+        :param str user: Username used to connect to CouchDB.
```

Comment:
Should we change these descriptions to be generic since the user can connect to CouchDB or Cloudant?